bpfd-operator/internal/bpfd-client-helpers.go

```diff
@@ -110,10 +119,167 @@ func LoadTLSCredentials(tlsFiles Tls) (credentials.TransportCredentials, error)
 	return credentials.NewTLS(config), nil
 }
 
-func BuildBpfdLoadRequest(bpf_program_config_spec *bpfdiov1alpha1.BpfProgramConfigSpec, name string) (*gobpfd.LoadRequest, error) {
+// Structure from https://github.com/kubernetes/kubernetes/blob/master/pkg/credentialprovider/config.go#L39
+// ContainerConfigJSON represents ~/.docker/config.json file info
```

Comment:
Comment is not in `godoc` format:

```suggestion
// ContainerConfigJSON represents ~/.docker/config.json file info.
// Structure from https://github.com/kubernetes/kubernetes/blob/master/pkg/credentialprovider/config.go#L39
```